.gitignore

```diff
@@ -58,4 +58,10 @@ typings/
 .env
 
 /store/*
-!/store/.gitkeep
\ No newline at end of file
+!/store/.gitkeep
+
+# e2e test
+/bin/selenium-server-standalone-3.4.0.jar
+/bin/geckodriver
+/bin/chromedriver
```

Comment:
Is this '/bin/*' part necessary?
I thought it was not necessary in my environment.